urg_node:
  ros__parameters:
    serial_port: "/dev/hokuyo"
    laser_frame_id: "laser_frame"
    laser_topic: "/scan"
    angle_offset: -1.22173
